inc/class-s3-uploads-wp-cli-command.php

```diff
@@ -117,6 +117,9 @@ public function migrate_attachment_to_s3( $args, $args_assoc ) {
 
 		foreach ( $files as $file ) {
 			if ( file_exists( $path = $old_upload_dir['basedir'] . '/' . $file ) ) {
+				if ( is_dir( $path ) ) {
```

Comment:
I think it would be better to just not add `_wp_attached_file` to the array of files if it's not set, rather than the is_dir check here.